uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/staff-leave-report.html

```diff
@@ -287,7 +287,7 @@
                           </tr>
                         </thead>
                         <tbody>
-                          <tr ng-repeat="entry in report.sections.expired.data track by $index">
+                          <tr ng-repeat="entry in report.sections.expired.data | orderBy:'+expiry_date' track by $index">
```

Comment:
Sorting expired requests was not part of the original ticket, but it seemed like a similar issue that could come up in the future.